src/intervals/rounding.jl

```diff
@@ -187,7 +248,9 @@ function make_interval(T, expr1, expr2)
 end
 
 
-float(x::Interval) = make_interval(Float64, x)
+float(x::Interval) = Interval(convert(Float64,x.lo),convert(Float64,x.hi))
```

Comment:
I don't think this is right. There should be some directed rounding possibly needed.